monitor/data_monitoring.py

```diff
@@ -64,8 +51,16 @@ def __init__(
         # slack client is optional
         self.slack_client = SlackClient.create_slack_client(self.slack_token, self.slack_webhook)
         self._download_dbt_package_if_needed(force_update_dbt_package)
+        self.elementary_database_and_schema = self.get_elementary_database_and_schema()
```

Comment:
This should not run on the intitial.
It relevant only for alerts, and there is no need to run this when creating the report.

Reply:
Totally right!